cn_indian_payroll/cn_indian_payroll/overrides/payroll.py

```
@@ -3,16 +3,33 @@
 class CustomPayrollEntry(PayrollEntry):
-    # @frappe.whitelist()
-    # def custpayroll(self, value):
-    #     frappe.msgprint(value)
 
-    # def before_save(self):
-    #     frappe.msgprint(str(self.company))
-    @frappe.whitelist()
-    def payrollset(self,value1):
-        frappe.msgprint(str(value1))
 
+    def on_cancel(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `on_cancel`

Add return type annotation: `None`

(ANN201)